bitcoin/src/bip152.rs

```diff
@@ -284,7 +284,7 @@ impl Decodable for BlockTransactionsRequest {
                 // Since the number of indices ultimately represent transactions,
                 // we can limit the number of indices to the maximum number of
                 // transactions that would be allowed in a vector.
-                let byte_size = (nb_indexes as usize)
+                let byte_size = (nb_indexes)
```

Comment:
Hmm, it didn't complain about useless parentheses?

Reply:
We should remove this later. Merging this to get the CI rolling

Reply:
bah, I must be blind. Will fix, thanks.